src/GitLinkTask/GitLink.targets

```diff
@@ -10,6 +10,7 @@
         PdbFile="$(IntermediateOutputPath)$(TargetName).pdb"
         Method="$(GitLinkMethod)"
         SkipVerify="$(GitLinkSkipVerify)"
+        IndexAllDepotFiles="$(GitLinkIndexAllDepotFiles)"              
```

Comment:
See my other comment, since @AArnott correctly identifies a (possible) performance issue, let's keep this "explicit" for now where people will either have to define it in their proj file or enable it using the -a switch.